src/containers/intake/schemas/IntakeSchemas.js

```diff
@@ -0,0 +1,325 @@
+      type: 'object',
+      title: '',
+      properties: {
+        [getEntityAddressKey(0, STAFF, OPENLATTICE_ID_FQN)]: {
+          type: 'string',
+          title: 'Staff Member',
+          enum: [],
+          enumNames: [],
+        },
+      },
+      required: [getEntityAddressKey(0, STAFF, OPENLATTICE_ID_FQN)]
+    },
+    [getPageSectionKey(1, 5)]: {
+      type: 'object',
+      title: '',
+      properties: {
+        [getEntityAddressKey(0, FORM, DATETIME_ADMINISTERED)]: {
+          type: 'string',
+          title: 'Date of Form Completion',
+          format: 'date',
+          default: DateTime.fromISO(currentDateTime).toISODate()
+        },
+        [getEntityAddressKey(0, FORM, NAME)]: {
+          type: 'string',
+          title: 'Form Name',
+          default: INTAKE_FORM
+        },
+        [getEntityAddressKey(0, STATUS, PropertyTypes.STATUS)]: {
+          type: 'string',
+          title: 'Status',
+          default: INTAKE
+        },
+        [getEntityAddressKey(0, STATUS, EFFECTIVE_DATE)]: {
+          type: 'string',
+          title: 'Intake Date',
+          default: currentDateTime
+        },
+      }
+    },
+  },
+};
+
+const uiSchema = {
+  [getPageSectionKey(1, 1)]: {
+    'ui:disabled': true,
+    classNames: 'column-span-12 grid-container',
+    [getEntityAddressKey(0, PEOPLE, GIVEN_NAME)]: {
+      classNames: 'column-span-4',
+    },
+    [getEntityAddressKey(0, PEOPLE, MIDDLE_NAME)]: {
+      classNames: 'column-span-4',
+    },
+    [getEntityAddressKey(0, PEOPLE, SURNAME)]: {
+      classNames: 'column-span-4',
+    },
+    [getEntityAddressKey(0, PEOPLE, DOB)]: {
+      classNames: 'column-span-4'
+    },
+    [getEntityAddressKey(0, PEOPLE, RACE)]: {
+      classNames: 'column-span-4'
+    },
+    [getEntityAddressKey(0, PEOPLE, ETHNICITY)]: {
+      classNames: 'column-span-4'
+    },
+  },
+  [getPageSectionKey(1, 2)]: {
+    'ui:disabled': true,
+    classNames: 'column-span-12',
+    'ui:options': {
+      addable: false,
+      orderable: false,
+      removable: false,
+    },
+    items: {
+      classNames: 'grid-container',
+      [getEntityAddressKey(-1, PEOPLE, GIVEN_NAME)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PEOPLE, MIDDLE_NAME)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PEOPLE, SURNAME)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PEOPLE, DOB)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PERSON_DETAILS, GENDER)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PERSON_DETAILS, PRONOUN)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PEOPLE, RACE)]: {
+        classNames: 'column-span-4'
+      },
+      [getEntityAddressKey(-1, PEOPLE, ETHNICITY)]: {
+        classNames: 'column-span-4'
+      },
+    }
+  },
+  [getPageSectionKey(1, 3)]: {
```

Comment:
yes, i think so! i'm going to hold off on constantifying these for right now, just because it's super time consuming and i'd rather do that once i've got these big pieces of the app in place.